examples/specs/normalized/trellis_row_column_normalized.vl.json

```diff
@@ -23,6 +23,14 @@
       "y": {
         "field": "Miles_per_Gallon",
         "type": "quantitative"
+      },
+      "color": {
+        "field": "Cylinders",
+        "type": "nominal"
+      },
+      "shape": {
```

Comment:
Look in the example above. It makes it clear what is happening. This is not a great visualization but a great test.